lib/cloudregister/registerutils.py

```diff
@@ -44,6 +44,7 @@
 REGISTRATION_DATA_DIR = '/var/cache/cloudregister/'
 REGISTERED_SMT_SERVER_DATA_FILE_NAME = 'currentSMTInfo.obj'
 RMT_AS_SCC_PROXY_MARKER = 'rmt_is_scc_proxy'
+TIME_CACHED = 60 * 5  # 5 min in seconds
```

Comment:
Nitpic: May 'CACHE_EXPIRATION_TIME' be more descriptive name? :thinking:

Reply:
What is the GCE JWT token expiration time? And yes a more descriptive name is in order.

Reply:
GCE JWT token for our rmt-verification is [300](https://gitlab.suse.de/public-cloud-team/rmt-server-verify-instance-gce/-/blob/master/gce.rb#L143)